app/controllers/concerns/idv_session.rb

```diff
@@ -11,6 +11,8 @@ def confirm_idv_needed
               decorated_sp_session.requested_more_recent_verification? ||
               idv_session_user.reproof_for_irs?(service_provider: current_sp)
 
+    return if !idv_session.personal_key_acknowledged && idv_session.personal_key.present?
```

Comment:
Nitpick: I think this reads better with the conditions reversed. We only want to check if the personal_key has been acknowledged if it's still in the session.
```suggestion
    return if idv_session.personal_key.present? && !idv_session.personal_key_acknowledged
```